files/galaxy/tpv/tools.yml

```diff
@@ -284,6 +284,10 @@ tools:
       - if: helpers.tool_version_gte(tool, '2.3.1+galaxy2')
         params:
           singularity_run_extra_arguments: "--env ALPHAFOLD_DB=/data/db/databases/alphafold_databases,ALPHAFOLD_USE_GPU=False"
+        execute: |
```

Comment:
Do we have to change this `ALPHAFOLD_USE_GPU=False` to `True`?

Reply:
I don't know, I saw this in the wrapper:
`-use_gpu_relax=\${ALPHAFOLD_USE_GPU:-True`

Reply:
Good point, we could also omit the `ALPHAFOLD_USE_GPU` entirely in the updated parameter, because the tool wrapper sets it to `True` by default, but this is more explicit

Reply:
Cool, looks fine. Thanks! Let's try it.